src/forms/signIn.jsx

```diff
@@ -9,17 +9,36 @@ import Box from "@mui/material/Box";
 import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
 import Typography from "@mui/material/Typography";
 import Container from "@mui/material/Container";
+import { useNavigate } from "react-router-dom";
 import { ThemeProvider } from "@mui/material/styles";
 import theme from "../theme";
+import { useDispatch } from "react-redux";
+import { signIn } from "../services/tableDataServices";
+import { sign_In } from "../redux/actions/action";
+import { ToastContainer, toast } from "react-toastify";
 
 export default function SignIn({ handleChange }) {
-  const handleSubmit = (event) => {
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const handleSubmit = async (event) => {
     event.preventDefault();
-    const data = new FormData(event.currentTarget);
-    console.log({
-      email: data.get("email"),
-      password: data.get("password")
-    });
+    const newUser = {
+      email: email,
+      password: password
+    };
+    const userData = await signIn(newUser);
+    dispatch(sign_In(userData));
+    if (userData?.metadata?.status === "SUCCESS") {
+      toast(userData?.metadata?.message);
```

Comment:
don't require for success it will not show because it will navigate to next page.